src/baseConfig.js

```diff
@@ -11,6 +11,8 @@ export class BaseConfig {
 
   constructor() {
     this._current = {
+      endpoint: null, // Endpoint to use for auth requests (local only). Null uses HttpClient.
```

Comment:
You shouldn't touch client. I use that internally. You _could_ use it, but you're not supposed to.